src/database/models/topic-assessment-info.ts

```diff
@@ -94,7 +94,10 @@ TopicAssessmentInfo.init({
         field: 'topic_assessment_info_max_graded_attempts_per_version',
         type: DataTypes.INTEGER,
         allowNull: false,
-        defaultValue: 0
+        defaultValue: 1,
+        validate: {
```

Comment:
@doombot-exe on practice exams can't this be 0 or -1?

Reply:
yes, but the default should be 1 attempt - as that is likely the most commonly preferred setting